broadcast/src/linked/mod.rs

```diff
@@ -0,0 +1,663 @@
+//! Secure, ordered broadcast protocol with linked messages for consistent network
```

Comment:
I think we can do a punchier/more succinct summary (this sounds a little word soup/AI).

I'm thinking something like:

```
Ordered, reliable broadcast across multiple, dynamic participants.
```

You can follow that with a `\n` and do a longer description but would make it more pointed than what is here.